### ISSUE TITLE
make docker-build error: can't cd to crosstool-ng-1.24.0

### ISSUE THREAD

**Issue by MarcBoule** [VCVRack/rack-plugin-toolchain#13]
I tried the updated v2 toolchain (423f9c53), and on `make docker-build` I'm getting an error at the `make toolchain-win` step. Is it possible a directory is not well specified in the makefile, since I think the directory it should point at is plainly `crosstool-ng`, since that looks like the directory I see in the folder, but the makefile seems to be telling it to cd into `crosstool-ng-1.24.0`, which does not seem to exist. I think I got the same error when trying the non-docker option.

```
d crosstool-ng-1.24.0 && make -j 8
/bin/sh: 1: cd: can't cd to crosstool-ng-1.24.0
make: *** [Makefile:33: /home/build/rack-plugin-toolchain/local/bin/ct-ng] Error 2
The command '/bin/sh -c JOBS=$JOBS make toolchain-win' returned a non-zero code: 2
make: *** [Makefile:230: docker-build] Error 2

```

**Comment by cschol** [comment]
Looking into it. I think it might have been a merge mistake on my part.

**Comment by cschol** [comment]
Fixed in e341661e57c84a6a119b5722b367619dde8d6622 .
